### DISCUSSION TITLE
Unable to Search Citizen Complaints in Employee Domain - Error Message Received

### DISCUSSION THREAD

**Discussion by Pooja25102000** [upyog/UPYOG#1118] [Q&A]
I'm facing an issue in the employee domain of UpYog where I'm unable to search for complaints submitted by citizens. Whenever I try to perform a search, I receive an error message stating "Something went wrong." I've attached screenshots for reference.
Steps to Reproduce:
Log in to the employee domain of UpYog.
Navigate to the search functionality for citizen complaints.
Enter search criteria and attempt to perform a search.
Expected Behavior:
I expect to be able to search for complaints submitted by citizens and view relevant information without encountering any errors.
Actual Behavior:
Upon attempting to search, I receive an error message stating "Something went wrong," and no search results are displayed.
Screenshots:
Screenshot : Error message received when attempting to search.
Additional Information:
This issue occurs consistently and is not limited to specific search criteria.
I've tried clearing browser cache and cookies, but the issue persists.
Other functionalities within the employee domain seem to be working fine; the problem appears to be specific to searching citizen complaints.
Any insights or suggestions on how to troubleshoot and resolve this issue would be greatly appreciated.
Thank you in advance for your assistance!
![Screenshot (10)](https://github.com/upyog/UPYOG/assets/166795686/4f92a9ba-bd91-41bd-9bb9-1a9023c60b9a)

**Comment by vinothrallapalli-eGov** [comment]
It seems the page is not getting the required data to render. Please check the API responses in the network tab, if any API is failing, check why failing and fix those issue then it will work.
